test/actions/Test_Actions.php

```diff
@@ -1498,5 +1498,71 @@ public function cannot_set_empty_actionid()
         // An empty ActionID
         $action->setActionID('');
     }
+
+    /**
+     * @test
+     */
+    public function can_update_config()
+    {
+        $number = 9876;
+        $writeCreate = array( implode("\r\n", array(
+            'action: UpdateConfig',
+            'actionid: 1432.123',
+            'srcfilename: sip.conf',
+            'dstfilename: sip.conf',
+            'action-000000: NewCat',
+            'cat-000000: '.$number,
+            'action-000001: Append',
+            'cat-000001: '.$number,
+            'var-000001: username',
+            'value-000001: test',
+            'action-000002: Append',
+            'cat-000002: '.$number,
+            'var-000002: secret',
+            'value-000002: secret',
+            ''
+        )) );
+
+        $actionCreate = new \PAMI\Message\Action\UpdateConfigAction();
+
+        $actionCreate->setSrcFilename('sip.conf');
+        $actionCreate->setDstFilename('sip.conf');
+        
+        $actionCreate->setAction('NewCat');
+        $actionCreate->setCat($number);
+
+        $actionCreate->setAction('Append');
+        $actionCreate->setCat($number);
+        $actionCreate->setVar('username');
+        $actionCreate->setValue('test');
+
+        $actionCreate->setAction('Append');
+        $actionCreate->setCat($number);
+        $actionCreate->setVar('secret');
+        $actionCreate->setValue('secret');
+
+        $client = $this->_start($writeCreate, $actionCreate);
+        
+        $writeDelete = array( implode("\r\n", array(
+            'action: UpdateConfig',
+            'actionid: 1432.123',
+            'srcfilename: sip.conf',
+            'dstfilename: sip.conf',
+            'Reload: yes',
+            'action-000000: DelCat',
+            'cat-000000: '.$number,
```

Comment:
The test fails on these asserts. Please run the tests on your box and let me know when you're sure the tests pass as expected.